### PR TITLE
Minutes for Meeting 2021-12-09

### PR DESCRIPTION
**Pull request by sthagen**
~Links to agenda and previous minutes draft not yet filled in~

This is now the post-meeting minutes document.

### CODE REVIEW

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+* Approved Teleconferences (Tuesdays at 08:00 PT / 16:00 UTC for 1.5 hours)
```
Comment (katrinaoneil) [inline meeting_minutes/211209_SARIF_TC_49.md:63]: Tuesdays -> Thursdays

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  ```
+  Januar 13
```
Comment (katrinaoneil) [inline meeting_minutes/211209_SARIF_TC_49.md:65]: Januar -> January

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+| First Name | Last Name | Company          | Role(s)                 |
+| :--------- | :-------- | :--------------- | :---------------------- |
+| Aditya     | Sharad    | Microsoft        | Member
+| Chad       | Bentz     | defi SOLUTINS    | Public Participant
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| Katrina    | O'Neil    | Micro Focus      | Voting member           |
+  Nathan     | Baird     | Microsoft        | Member
+| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
+| Stefan     | Hagen     | Individual       | Secretary, taking notes |
```
Comment (eddynaka) [inline meeting_minutes/211209_SARIF_TC_49.md:16]:
```suggestion
| :--------- | :-------- | :--------------- | :---------------------- |
| Aditya     | Sharad    | Microsoft        | Member                  |
| Chad       | Bentz     | defi SOLUTINS    | Public Participant      |
| David      | Keaton    | Individual       | Chair                   |
| Eddy       | Nakamura  | Microsoft        | Voting member           |
| Katrina    | O'Neil    | Micro Focus      | Voting member           |
|  Nathan     | Baird     | Microsoft        | Member                 |
| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
| Stefan     | Hagen     | Individual       | Secretary, taking notes |
```

**Review by katrinaoneil — APPROVED** [review]

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+
+* ACTION David: Find out how to upload content to wikipedia (#507)
+* ACTION Michael: Reach out to contacts at Google and Uber on the Code Insights Protocol topic 
+* ACTION Michael: Reach out to ParaSoft
+* ACTION Michael: Will author and check in the roadmap / list of in- and out-of-scope features
+
+
+# 6. Next Meeting
+
+Januay 13, 2022 / 08:00-09:30 PT / 16:00-17:30 UTC
```
Comment (eddynaka) [inline meeting_minutes/211209_SARIF_TC_49.md:167]:
```suggestion
January 13, 2022 / 08:00-09:30 PT / 16:00-17:30 UTC
```

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+# 7. Adjournment TODO
```
Comment (katrinaoneil) [inline meeting_minutes/211209_SARIF_TC_49.md:169]: remove TODO?

File: meeting_minutes/211209_SARIF_TC_49.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+
+# 3. Discussion
+
+## 3.1 Review recruitment effort to complete the technical committee
+
+* None
+
+## 3.2 Review status on finalizing SARIF 2.1 errata
+
+* Shall the TC approve the combination of these documents as Committee Specification Draft Errata?
```
Comment (dmk42) [inline meeting_minutes/211209_SARIF_TC_49.md:78]: I am submitting the request to OASIS, and they have asked that the exact title appear in the minutes.  I suggest adding the following line before the web links.

Title:  Static Analysis Results Interchange Format (SARIF) Version 2.1.0 Errata CSD01

Comment (sthagen) [reply meeting_minutes/211209_SARIF_TC_49.md:78]: Hope the added change does accomplish this.

**Review by dmk42 — APPROVED** [review]
Thanks, Stefan!  I have now submitted the request to OASIS for the 15-day public review of the Errata.

**Pull request merged**